packages/perseus-editor/src/components/locked-figures-section.tsx

```diff
@@ -92,20 +92,31 @@ const LockedFiguresSection = (props: Props) => {
 
     return (
         <View>
-            {figures?.map((figure, index) => (
-                <LockedFigureSettings
-                    expanded={expandedStates[index]}
-                    onToggle={(newValue) => {
-                        const newExpanded = [...expandedStates];
-                        newExpanded[index] = newValue;
-                        setExpandedStates(newExpanded);
-                    }}
-                    key={`${uniqueId}-locked-${figure}-${index}`}
-                    {...figure}
-                    onChangeProps={(newProps) => changeProps(index, newProps)}
-                    onRemove={() => removeLockedFigure(index)}
-                />
-            ))}
+            {figures?.map((figure, index) => {
+                if (figure.type === "circle") {
+                    // TODO(LEMS-1941): Implement circle locked figure settings.
+                    // Remove this block once circle locked figure settings are
+                    // implemented.
+                    return;
+                }
```

Comment:
**nit of all nits**: You could use `.filter()` here to remove any circles and then it would yield a less disruptive change here (to revert, you just remove the `.filter()`).  

I say this in full realization that this code will be reverted in the very near future.  :)   But, thinking more "functionally" by chaining array functions might be useful for other things.  

```suggestion
            {figures?
                // TODO(LEMS-1941): Implement circle locked figure settings.
                // Remove this block once circle locked figure settings are
                // implemented.
                .filter(f => f.type !== "circle")
                .map((figure, index) => {
```